dashboard/src/api/hooks/usePostVersion.ts

```diff
@@ -0,0 +1,32 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { postProjectRequest } from '../../interfaces/projects';
+import { GetVersionsResponse } from '../../interfaces/versions';
+import client from '../client';
+import '../mocks/usePostVersion.mock';
+
+const postVersion = async (projectId: number, request: postProjectRequest) => {
+  const response = await client.post<GetVersionsResponse>(
+    `/api/v1/projects/${projectId}/versions`,
```

Comment:
make "/api/v1/projects" as an exportable constants, because it is used different places.

Reply:
we tried to add `/api/v1/` to the baseURL in axios setup but then we came up the a conclusion that our current way of handling it is easier to understand and keep the same both in mocked and non-mocked environments